### PR TITLE
Translate to Indonesian

### PR DESCRIPTION
**Pull request by franzaurus**
Hi, [@sergeicodes](https://github.com/sergeicodes).

Thank you for building this cool project. I’m from Indonesia and I’d like to contribute to the Indonesian translation of your Accessibility Myths website. Your website got me curious about the people with disability in my country. It turns out that based on [this 2017 report](https://www.monash.edu/__data/assets/pdf_file/0003/1107138/Disability-in-Indonesia.pdf), _"there are at least 10 million people with some form of disability. This represents 4.3% of the population, based on the latest census which almost certainly understates its prevalence"._

That’s quite a figure. I just hope this meager contribution can help them navigate the web better.

Again, thank you for initiating this project.

-Frans

### CODE REVIEW

File: id/index.html [hunk 1]
```diff
@@ -0,0 +1,371 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Accessibility Myths</title>
+    <link rel="stylesheet" href="./css/styles.css" />
```
Comment (sergeicodes) [inline id/index.html:8]: @franzaurus please update `href` to `/css/styles.css` so styles are displayed properly.

File: id/index.html [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="es" href="/es/">Español</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="fr" href="/fr/">Français</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="id" href="/id/">Bahasa Indonesia</a> 
+           </li>
```
Comment (sergeicodes) [inline id/index.html:49]: @franzaurus please add a link to Bahasa Indonesia to all languages (`fr/index.html`, `de/index.html`, etc.) 

image

File: id/index.html [hunk 3]
```diff
@@ -0,0 +1,371 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Accessibility Myths</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link rel="icon" type="image/x-icon" href="./favicon.ico" />
```
Comment (sergeicodes) [inline id/index.html:9]: @franzaurus please update `href` to `/favicon.ico` so the icon is displayed properly.

**Review by sergeicodes — CHANGES_REQUESTED** [review]
@franzaurus thanks for translating! Please read comments I added, once they are handled, we can release Bahasa Indonesia language.

File: id/index.html [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+        </div>
+      </div>
+    </main>
+
+    <footer>
+      <div class="content">
+        <p>
+          © 2024
+          <a target="_blank" href="https://www.sergeikriger.com/">Sergei Kriger</a>. All rights reserved.
+        </p>
```
Comment (sergeicodes) [inline id/index.html:360]: @franzaurus could you please add some info about you (link to your site, github profile or anything you want) here in the footer like in other translations.

**Comment by sergeicodes** [comment]
@franzaurus please ignore my comments, I'll do everything myself.

**Comment by franzaurus** [comment]
> @franzaurus please ignore my comments, I'll do everything myself.

Thank you for releasing the Indonesian translation!

**Pull request merged**